lib/tasks/mastodon.rake

```diff
@@ -521,12 +535,53 @@ namespace :mastodon do
 
           prompt.ok "You can login with the password: #{password}"
           prompt.warn 'You can change your password once you login.'
+        else
+          prompt.warn 'Okay.'
+        end
+
+        prompt.say "\n"
+
+        blocklist = prompt.select("Would you like to import an initial domain blocklist for your instance?\n") do |menu|
+          menu.enum '.'
+
+          domain_blocklists.each do |list|
+            menu.choice "#{list[:name]} - website: #{list[:website]}", { url: list[:csv], name: list[:name] }
+          end
+          menu.choice 'No, skip setting up an initial blocklist', nil
         end
+
+        prompt.say "\n"
+
+        if db_connection_works && blocklist.present?
+          prompt.say "Starting import of #{blocklist[:name]} blocklist...\n"
+
+          cmd_env = env.transform_values(&:to_s).merge({ 'RAILS_ENV' => 'production' })
+          cmd = "./bin/tootctl domains blocklist import \"#{blocklist[:name]}\" \"#{blocklist[:url]}\""
```

Comment:
For some reason, passing the name & url via `Open3.popen2e(cmd_env, cmd, ...args)` didn't work, as it expects `./bin/tootctl` to be in PATH, despite it being a relative command, not a system command. I'm not a fan of this, but there didn't seem to be a nicer way that I could find. Theoretically it could be a command injection if the `domain_blocklists` array contains something nasty.. but I'm kinda hoping that couldn't be the case.

Reply:
Fixed by using `tty-command`